ChaosMod/Components/Globals.h

```diff
@@ -0,0 +1,76 @@
+#pragma once
+
+#include "Components/Component.h"
+
+#include "Memory/Hooks/ScriptThreadRunHook.h"
+
+#include <map>
+#include <vector>
+
+enum GlobalPatternIdiom : uint8_t
```

Comment:
```suggestion
enum class GlobalPatternIdiom : uint8_t
```
To not pollute the global namespace as much